packages/teamviewer-deb/teamviewer-deb.pacscript

```diff
@@ -0,0 +1,13 @@
+name="teamviewer-deb"
+pkgname="teamviewer"
+gives="teamviewer"
+version="15.34.4"
+repology=("project: teamviewer")
+replace="${gives}"
+breaks="${gives} ${gives}-bin ${gives}-git ${gives}-app ${gives}-deb"
+depends="libqt5x11extras5 libqt5quickcontrols2-5 libqt5webengine5 libqt5svg5 hicolor-icon-theme"
```

Comment:
Are these already included in the deb?

Reply:
I just checked and looks like they are not included.